src/libzippp.cpp

```diff
@@ -78,7 +78,25 @@ int ZipEntry::readContent(std::ostream& ofOutput, ZipArchive::State state, libzi
    return zipFile->readEntry(*this, ofOutput, state, chunksize);
 }
 
-ZipArchive::ZipArchive(const string& zipPath, const string& password) : path(zipPath), zipHandle(nullptr), zipSource(nullptr), mode(NOT_OPEN), password(password) {
+ZipArchive::ZipArchive(const string& zipPath, const string& password, Encryption encryptionMethod) : path(zipPath), zipHandle(NULL), zipSource(NULL), mode(NOT_OPEN), password(password){
+    switch(encryptionMethod){
+        case Encryption::AES_128:
+            this->encryptionMethod=ZIP_EM_AES_128;
+            break;
+        case Encryption::AES_192:
+            this->encryptionMethod=ZIP_EM_AES_192;
+            break;
+        case Encryption::AES_256:
+            this->encryptionMethod=ZIP_EM_AES_256;
+            break;
+        case Encryption::TRAD_PKWARE:
+            this->encryptionMethod=ZIP_EM_TRAD_PKWARE;
+            break;
+        case Encryption::NONE:
+        default:
+            this->encryptionMethod=ZIP_EM_NONE;
+            break;
```

Comment:
Please, respect format:
- space after closing parenthesis
- space each side of `=`